cmake/pkgconfig.in

```diff
@@ -6,6 +6,6 @@ Name: libOSRM
 Description: Project OSRM library
 Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
-Libs: -L${libdir} -losrm
+Libs: -L${libdir} -losrm @CMAKE_EXE_LINKER_FLAGS@
```

Comment:
Can this be different than
- CMAKE_MODULE_LINKER_FLAGS
- CMAKE_SHARED_LINKER_FLAGS
- CMAKE_STATIC_LINKER_FLAGS

and what do we have to report?